README.md

```diff
@@ -6,6 +6,13 @@
 lightweight RTOS-based platform to support low-end IoT devices.  
 Please find project details like **APIs**, **Specification** and **Long-term Goals** on our [Tizen Site](https://source.tizen.org/documentation/tizen-rt).
 
+## Contents
+
+> [Quick Start](#quick-start)  
```

Comment:
@lokeshbv Thank you for reveiwing. But I don't think indexing is helpful and meaningful for contents. I want to index with number when full sequence should be executed and sequence has an important meaning.